cli/lib/kontena/cli/nodes/labels/remove_command.rb

```diff
@@ -12,7 +12,7 @@ class RemoveCommand < Kontena::Command
     def execute
       node = client.get("nodes/#{current_grid}/#{node_id}")
       data = { labels: Array(node['labels']).reject {|label| label_list.include?(label) } }
-      client.put("nodes/#{current_grid}/#{node['id']}", data)
+      client.put("nodes/#{current_grid}/#{node['name']}", data)
```

Comment:
There is actually one edge-case where `nodes/#{node['id']}` works but `nodes/#{current_grid}/#{node['name']}` won't: if it's a new node that just connected with a grid token, and it hasn't updated its node_info yet.